frontend/src/Editor/QueryManager/QueryEditors/Stripe.jsx

```diff
@@ -96,20 +98,36 @@ class Stripe extends React.Component {
     this.props.optionsChanged(newOptions);
   };
 
-  renderOperationOption = (props, option, snapshot, className) => {
-    return (
-      <button {...props} className={className} type="button">
+  removeParam = (paramType, paramName) => {
+    const options = this.state.options;
+    const newOptions = JSON.parse(JSON.stringify(options));
+    options[paramName] = undefined;
```

Comment:
I think this should be `newOptions` ?
```suggestion
    newOptions[paramName] = undefined;
```